Documentation/network/servicemesh/tls-cert.rst

```diff
@@ -32,11 +32,11 @@ Create TLS Certificate and Private Key
         .. code-block:: shell-session
 
             $ helm repo add jetstack https://charts.jetstack.io
-            $ helm install cert-manager jetstack/cert-manager --version v1.10.0 \
+            $ helm install cert-manager jetstack/cert-manager --version v1.15.1 \
```

Comment:
it seems as this command is failing with the following error whereas the previous command worked without an issue.

```
❯ helm install cert-manager jetstack/cert-manager --version v1.15.1 \
                --namespace cert-manager \
                --create-namespace \
                --set crds.enabled=true \
                --set config.enableGatewayAPI=true
Error: INSTALLATION FAILED: execution error at (cert-manager/templates/controller-config.yaml:2:38): .Values.config.apiVersion must be set !
```

It looks like we need wait for https://github.com/cert-manager/cert-manager/pull/7126 being released or add the missing helm values too.

```
--set config.apiVersion="controller.config.cert-manager.io/v1alpha1" \
--set config.kind="ControllerConfiguration" \
```

Reply:
Thank you for testing out the changes @mhofstetter, I appreciate the attention to detail there.